app/database/models/server/category.ts

```diff
@@ -32,10 +32,10 @@ export default class CategoryModel extends Model implements CategoryInterface {
     /** associations : Describes every relationship to this table. */
     static associations: Associations = {
 
-        /** A CATEGORY has a 1:N relationship with CHANNEL. A CATEGORY can possess multiple channels */
+        /** A CATEGORY has a 1:N relationship with CATEGORY_CHANNEL. A CATEGORY can possess multiple channels */
```

Comment:
I wonder if we should word it like:
`A CHANNEL can belong to several CATEGORY, and a CATEGORY posses multiple channels (N:N relationship). We use the intermediate table CATEGORY_CHANNEL for this relationship`
(Not my best prose, but I think you get the message)